.github/workflows/release.yml

```diff
@@ -12,7 +12,7 @@ jobs:
       - uses: actions/checkout@v3
       - uses: actions/setup-node@v3
         with:
-          node-version: 18
+          node-version: 20.11
```

Comment:
The node version is updated according to their documentation.

![image](https://github.com/storyblok/storyblok-cli/assets/1240591/83cf1181-3718-4370-bde7-b90aeb048a76)

Another approach would be to fix a version (using the last successful version) to it and to run:

`npx semantic-release@22.0.12`
This way I think it would be ok to keep the node in version 18. 🙌